README.md

```diff
@@ -40,22 +40,31 @@ snowdrop	    https://snowdrop.github.io/helm
 
 ## Usage
 
-To use one of the available charts, part of the snowdrop repository such as `spring-boot-example-app`, you first need to generate a chart directory using the following command:
+To use one of the available charts, part of the snowdrop repository such as `spring-boot-example-app`, you first need to generate a chart directory in repository folder using the following command:
 
 ```console
+$ cd repository
```

Comment:
Why the repository folder?
The idea of this section was to explain how to use the Helm repository and one of its dependencies, not how to add a new chart in the repository